deployments/polygon/usdc/configuration.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "Compound USDC (Polygon)",
+  "symbol": "cUSDCv3",
+  "baseToken": "USDC",
+  "baseTokenAddress": "0x2791bca1f2de4661ed88a30c99a7a9449aa84174",
+  "baseTokenPriceFeed": "0xfE4A8cc5b5B2366C1B58Bea3858e81843581b2F7",
+  "borrowMin": "100e6",
+  "storeFrontPriceFactor": 0.5,
+  "targetReserves": "5000000e6",
+  "rates": {
+    "supplyKink": 0.8,
+    "supplySlopeLow": 0.0325,
+    "supplySlopeHigh": 0.4,
+    "supplyBase": 0,
+    "borrowKink": 0.8,
+    "borrowSlopeLow": 0.035,
+    "borrowSlopeHigh": 0.25,
+    "borrowBase": 0.015
+  },
+  "tracking": {
+    "indexScale": "1e15",
+    "baseSupplySpeed": "0e15",
+    "baseBorrowSpeed": "0e15",
+    "baseMinForRewards": "1000000e6"
+  },
+  "assets": {
+    "WETH": {
+      "address": "0x7ceb23fd6bc0add59e62ac25578270cff1b9f619",
+      "priceFeed": "0xF9680D99D6C9589e2a93a78A04A279e509205945",
+      "decimals": "18",
+      "borrowCF": 0.825,
+      "liquidateCF": 0.895,
+      "liquidationFactor": 0.95,
+      "supplyCap": "500000e18"
+    },
+    "WBTC": {
+      "address": "0x1bfd67037b42cf73acf2047067bd4f2c47d9bfd6",
+      "priceFeed": "0xDE31F8bFBD8c84b5360CFACCa3539B938dd78ae6",
+      "decimals": "8",
+      "borrowCF": 0.70,
+      "liquidateCF": 0.77,
+      "liquidationFactor": 0.95,
+      "supplyCap": "500000e18"
+    },
+    "WMATIC": {
```

Comment:
so the plan to launch on Polygon without `WMATIC` as a collateral asset hit a snag: having a functioning Bulker requires passing in a `wrappedNativeAsset` that is a collateral asset in the protocol.

so if you create an instance of Bulker with `WMATIC` as the `wrappedNativeAsset`, but `WMATIC` is not a collateral asset, then attempts to use the `ACTION_SUPPLY_NATIVE_ASSET` and `ACTION_WITHDRAW_NATIVE_ASSET` actions will fail.

our options seem to be:

- launch on Polygon without Bulker; update the UI to work without Bulker
- launch with a Bulker where the `ACTION_SUPPLY_NATIVE_ASSET` and `ACTION_WITHDRAW_NATIVE_ASSET` will revert; update scenarios to exclude the Polygon deployment from scenarios that test that functionality
- launch with WMATIC; add cWMATICv3 later (ending up with both as collateral assets)
- launch cWMATICv3, followed by cUSDCv3 (with cWMATICv3 as collateral asset)

Reply:
discussed during Parking lot.

it sounds like we're comfortable deploying with a Bulker that has a few actions that revert. when we want to have a Polygon Bulker that is capable of wrapping MATIC via a cWMATICv3 deployment, we'll need to create a new Bulker that takes in another Comet instance as a wrapper asset (instead of a WETH-style wrapper asset).